options.html

```diff
@@ -12,6 +12,9 @@
     <link rel="stylesheet" href="options.css">
   </head>
   <body>
+    <a href="https://github.com/nrchandan/mult-table-chrome-ext">
+      <img loading="lazy" width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_left_red_aa0000.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1">
+    </a>
```

Comment:
Thank you for adding this! Could you try to add a pure CSS ribbon instead of image? Meanwhile, let me merge this.